### PR TITLE
set root privilege when installing npm update and change npm version to 2.7.1

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

File: armv7hf/0.10/Dockerfile [hunk 1]
```diff
@@ -2,12 +2,12 @@
 FROM resin/armv7hf-buildpack-deps:jessie
 
 ENV NODE_VERSION 0.10.36
-ENV NPM_VERSION 2.5.0
+ENV NPM_VERSION 2.7.1
 
-RUN curl -SLO "http://resin-packages.s3.amazonaws.com/node/v$NODE_VERSION/node-v$NODE_VERSION-linux-armv7hf.tar.gz" \
+RUN curl -SLO "http://resin-packages.s3.amazonaws.com/node/v0.10.36/node-v0.10.36-linux-armv7hf.tar.gz" \
```
Comment (Page-) [inline armv7hf/0.10/Dockerfile]: Hard coded node version?

Comment (nghiant2710) [inline armv7hf/0.10/Dockerfile]: the node version is replaced in generating script only for this line.

Comment (Page-) [inline armv7hf/0.10/Dockerfile]: As far as I can tell from the generating script it shouldn't be?  It replaces `#{NODE_VERSION}` not `$NODE_VERSION`

Comment (petrosagg) [inline armv7hf/0.10/Dockerfile]: Yeah, something looks wrong here

Comment (nghiant2710) [inline armv7hf/0.10/Dockerfile]: This line comes from here: https://github.com/resin-io-library/docker-node/blob/master/generate-all-dockerfile.sh#L6-L7.
I did replace the node version and arch of this line in the script.
@petrosagg @Page-

**Pull request merged**